receiver/README.md

```diff
@@ -180,3 +180,37 @@ receivers:
             static_configs:
               - targets: ['localhost:9777']
 ```
+
+## <a name="vmmetrics"></a>VM Metrics Receiver
+**Only metrics are supported.**
+
+<Add more information - I'm lonely.>
+
+## <a name="zipkin"></a>Zipkin Receiver
+**Only traces are supported.**
+
+This receiver receives spans from Zipkin (V1 and V2) HTTP uploads and translates them into the internal span types that are then sent to the collector/exporters.
+
+Its address can be configured in the YAML configuration file under section "receivers", subsection "zipkin" and field "address".  The syntax of the field "address" is `[address|host]:<port-number>`.
+
+For example:
+
+```yaml
+receivers:
+  zipkin:
+    address: "127.0.0.1:9411"
```

Comment:
```suggestion
    endpoint: "127.0.0.1:9411"
```

Reply:
The conversation got shrunk but the zipkin documentation hasn't changed it was just moved around to be alphabetically ordered. The intent in this pr isn't to update anything other than the beginning of the oc receiver documentation.